.editorconfig

```diff
@@ -0,0 +1,30 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+# The JSON files contain newlines inconsistently
+[*.json]
+insert_final_newline = ignore
+
+# Minified JavaScript files shouldn't be changed
+[**.min.js]
+indent_style = ignore
+insert_final_newline = ignore
+
+# Makefiles always use tabs for indentation
+[{Makefile, makefile, GNUmakefile, Makefile.*}]
+indent_style = tab
+
+[*.md]
+trim_trailing_whitespace = true
+
+[*.sh]
+indent_style = space
```

Comment:
Suggestion: indent_style is already `space` for the root setting. setting it again for `sh` isn't needed.